src/Translator/doc/index.rst

```diff
@@ -0,0 +1,108 @@
+Symfony UX Translator
+=====================
+
+Symfony UX Translator is a Symfony bundle providing the same mechanism as `Symfony Translator`_
+in JavaScript with a TypeScript integration, in Symfony applications. It is part of `the Symfony UX initiative`_.
+
+The `ICU Message Format`_ is also supported.
+
+Installation
+------------
+
+Before you start, make sure you have `Symfony UX configured in your app`_.
+
+Install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-translator
+
+    # Don't forget to install the JavaScript dependencies as well and compile
+    $ npm install --force
+    $ npm run watch
+
+    # or use yarn
+    $ yarn install --force
+    $ yarn watch
+
+Usage
+-----
+
+When warming up the Symfony cache, all of your translations will be dumped as JavaScript into the ``var/translations/`` directory.
+For a better developer experience, TypeScript types definitions are also generated aside those JavaScript files.
+
+Then, you will be able to import those JavaScript translations in your assets.
+Don't worry about your final bundle size, only the translations you use will be included in your final bundle, thanks to the [tree shaking](https://webpack.js.org/guides/tree-shaking/).
+
+Configuring the default locale and locale fallbacks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, if the JavaScript translator is not configured, it will:
+- use the value of ``document.documentElement.lang`` or ``en`` (English) as the default locale
+- not use any locale fallbacks
+
+This step is optional, but if you want to configure the default locale and locale fallbacks for the JavaScript translator, you must use the Twig function ``initialize_js_translator()``:
+
+.. code-block:: twig
+
+    {# templates/base.html.twig #}
+
+    {% block javascripts %}
+        {# It will use `Locale::getDefault()` or `en` as default locale #}
+        {{ initialize_js_translator() }}
```

Comment:
Is this the only way to setup fallbacks? Could this be done as part of the cache:warmup (dump the fallbacks)?

Reply:
Currently this is the only way, but you're right I think we can configure fallbacks through `cache:warmup`.